app/src/main/java/com/example/triptracker/model/repository/ItineraryRepository.kt

```diff
@@ -118,57 +101,14 @@ open class ItineraryRepository {
       _itineraryList.add(itinerary)
     }
   }
-  //
-  //  // Update an itinerary in the database
-  //  /**
-  //   * Update an itinerary in the database
-  //   *
-  //   * @param itinerary Itinerary object to update Serialize the Itinerary object to a map and
-  // update
-  //   *   it in the database
-  //   */
-  //  fun updateItinerary(itinerary: Itinerary) {
-  //    val itineraryData =
-  //        hashMapOf<String, Any>(
-  //            "id" to itinerary.id,
-  //            "title" to itinerary.title,
-  //            "userMail" to itinerary.userMail,
-  //            "location" to
-  //                hashMapOf(
-  //                    "latitude" to itinerary.location.latitude,
-  //                    "longitude" to itinerary.location.longitude,
-  //                    "name" to itinerary.location.name),
-  //            "flameCount" to itinerary.flameCount,
-  //            "saves" to itinerary.saves, // Added saves field to Itinerary class
-  //            "clicks" to itinerary.clicks, // Added clicks field to Itinerary class
-  //            "numStarts" to itinerary.numStarts, // Added numStarts field to Itinerary class
-  //            "startDateAndTime" to itinerary.startDateAndTime,
-  //            "endDateAndTime" to itinerary.endDateAndTime,
-  //            "pinnedPlaces" to
-  //                itinerary.pinnedPlaces.map { pin ->
-  //                  hashMapOf(
-  //                      "latitude" to pin.latitude,
-  //                      "longitude" to pin.longitude,
-  //                      "name" to pin.name,
-  //                      "description" to pin.description,
-  //                      "image-url" to pin.image_url)
-  //                },
-  //            "description" to itinerary.description,
-  //            "route" to convertLatLngToMap(itinerary.route))
-  //    db.collection("itineraries")
-  //        .document(itinerary.id)
-  //        .set(itineraryData)
-  //        .addOnSuccessListener { Log.d(TAG, "Itinerary updated successfully") }
-  //        .addOnFailureListener { e -> Log.e(TAG, "Error updating itinerary", e) }
-  //  }
 
   /**
    * Add a new itinerary to the database
    *
    * @param itinerary Itinerary object to add Serialize the Itinerary object to a map and add it to
    *   the database
    */
-  suspend fun addNewItinerary(itinerary: Itinerary) {
+  fun addNewItinerary(itinerary: Itinerary) {
```

Comment:
Why is the suspend removed ?

Reply:
It was displayed as being redundant and unnecessary